lib/internal/Magento/Framework/Mail/Message.php

```diff
@@ -7,14 +7,25 @@
  */
 namespace Magento\Framework\Mail;
 
-class Message extends \Zend_Mail implements MessageInterface
+use Zend\Mime\Mime;
+use Zend\Mime\Part;
+
+/**
+ * @todo composition instead of inheritance for better testability
+ * - add a ZendMailDecorator interface with getZendMail() method for usage in \Magento\Framework\Mail\Transport
+ * @todo get rid of temporal coupling (setMessageType() + setBody())
+ * - deprecate setMessageType(), implement a HtmlMessage decorator instead
```

Comment:
Another advantage is that this resembles the old zf1 implementation, so 3rd party code is actually less likely to break even though we have to change an `@api` interface